script/bootstrap

```diff
@@ -19,51 +19,25 @@
 
 set -e
 
-PROJECT_ROOT_DIR="$(cd $(dirname "$0")/.. && pwd -P)"
-THIRD_PARTY_DIR="${PROJECT_ROOT_DIR}/third_party"
-DEFAULT_LOG_DIR="${PROJECT_ROOT_DIR}/script/log"
+source "$(cd $(dirname "$0") && pwd -P)/.utils"
```

Comment:
Why not simply `source .utils`?

Reply:
because `.utils` is an relative path, therefore script will search for `.utils` in directory, from which it is executed.